proposals/0002-root-signature-in-clang.md

```diff
@@ -402,26 +404,6 @@ registers are bound multiple times, or where there are multiple RootFlags
 entries, so subsequent stages should not assume that any given root signature in
 IR is valid.
 
-
-> **Open Question**: what should the named metadata be?  There's options I
-> think...
-
-Example for same root signature as above:
-
-```llvm
-; placeholder - update this once detailed design complete
-!directx.rootsignatures = !{!2}
-!2 = !{ptr @main, !3 }
-!3 = !{ !4, !5, !6, !7 } ; reference 4 root parameters
-!4 = !{ !"RootFlags", i32 1 } ; root flags, 1 is numeric value of flags
-!5 = !{ !"RootCBV", i32 0, i32 1, i32 0, i32 0 } ; register 0, space 1, 0 = visiblity, 0 = flags
-!6 = !{ !"StaticSampler", i32 1, i32 0, ... } ; register 1, space 0, (additional params omitted)
-!7 = !{ !"DescriptorTable", i32 0, !8, !9 } ;  0 = visibility, 2 ranges,!8 and !9
-!8 = !{ !"SRV", i32 0, i32 0, i32 -1, i32 0 } ; register 0, space 0, unbounded, flags 0
-!9 = !{ !"UAV", i32 5, i32 1, i32 10, i32 0 } ; register 5, space 1, 10 descriptors, flags 0
-```
```

Comment:
This example should stay here since it's part of proposed solutions.

Reply:
Updated.

Reply:
It looks like this section is still in the detailed design part rather than the proposed solution.